Cargo.toml

```diff
@@ -13,7 +13,7 @@ err-ctx = "0.2"
 fern = "0.5"
 hyper = "0.12"
 hyper-tls = "0.3"
-log = "0.4"
+log = "^0.4.8"
```

Comment:
As I understand it, this line should be equivalent to the old one? Or at least, should be equivalent after running `cargo update` - all this does is increase the minimum version, which I don't think we should need to do.

Reply:
Resolution conflict with another package that wanted 0.4.8. I have absolutely no idea why that happened though since I agree they should be compatible.

Reply:
I guess that make some sort of sense...? Cargo tries to keep multiple crates using similar versions if it can, but also won't update things implicitly. If you remove this line, does running `cargo update` again after doing that fix the conflict or does it still complain?